web-ng/src/app/components/layer-dialog/layer-dialog.component.html

```diff
@@ -24,7 +24,7 @@
 <div class="default-style">
   <div class="default-style-tag">Default style</div>
   <app-edit-style-button
-    [markerColor]="layer!.color || '#000'"
+    [markerColor]="layer!.color || '#ff9131'"
```

Comment:
Would it be possible to read this from this constant instead?
https://github.com/google/ground-platform/blob/4e8c99e193b8f6c1679113bab41e72c6dca97565/web-ng/src/app/components/layer-dialog/layer-dialog.component.ts#L47